### PR TITLE
Server installation docs updates

### PR DESCRIPTION
**Pull request by rosieyohannan**
UI text change, and reference IP as well as hostname. Hostname is AWS terminology, IP more inclusive for GCP users.

### CODE REVIEW

File: jekyll/_cci2/server-3-install-migration.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ In a terminal:
...
 . The 3.x instance will either need to be updated to point at the same storage bucket that the 2.19 instance used, or the data needs to be copied over to a new bucket.  The latter will ensure the 2.19 instance continues to work as expected, and so is the recommended approach if this migration is part of a test.
 
-NOTE: If a different hostname is being used in the 3.x environment, the GitHub webhooks will still be pointing to the hostname used in the 2.19 environment.  The easiest way to update this is to click *Stop Building* and then *Set Up Project*. After doing this, the contexts and environment variables associated with the project will still be present.
+NOTE: If a different hostname/IP is being used in the 3.x environment, the GitHub webhooks will still be pointing to the hostname used in the 2.19 environment.  The easiest way to update this is to click *Stop Building* and then *Set Up Project*. After doing this, the contexts and environment variables associated with the project will still be present.
```
Comment (catherinepope) [inline jekyll/_cci2/server-3-install-migration.adoc:75]:
```suggestion
NOTE: If a different hostname/IP is being used in the 3.x environment, the GitHub webhooks will still be pointing to the hostname used in the 2.19 environment. The easiest way to update this is to click *Stop Building* and then *Set Up Project*. After doing this, the contexts and environment variables associated with the project will still be present.
```

Comment (catherinepope) [reply jekyll/_cci2/server-3-install-migration.adoc:75]: Removed double space.

**Review by yugesh84 — APPROVED** [review]
Have one minor comment, but otherwise LGTM. Added server to ensure they are ok with this too.

**Review by ryanwohara — APPROVED** [review]

File: jekyll/_cci2/server-3-install-migration.adoc [hunk 2]
```diff
@@ -72,7 +72,7 @@ In a terminal:
...
 . The 3.x instance will either need to be updated to point at the same storage bucket that the 2.19 instance used, or the data needs to be copied over to a new bucket.  The latter will ensure the 2.19 instance continues to work as expected, and so is the recommended approach if this migration is part of a test.
 
-NOTE: If a different hostname is being used in the 3.x environment, the GitHub webhooks will still be pointing to the hostname used in the 2.19 environment.  The easiest way to update this is to click *Stop Building* and then *Set Up Project*. After doing this, the contexts and environment variables associated with the project will still be present.
+NOTE: If a different hostname/IP is being used in the 3.x environment, the GitHub webhooks will still be pointing to the hostname used in the 2.19 environment. The easiest way to update this is to click *Stop Building* and then *Set Up Project*. After doing this, the contexts and environment variables associated with the project will still be present.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-install-migration.adoc:75]:
```suggestion
NOTE: If a different hostname is being used in the 3.x environment, the GitHub webhooks will still be pointing to the hostname used in the 2.19 environment. The easiest way to update this is to click *Stop Building* and then *Set Up Project*. After doing this, the contexts and environment variables associated with the project will still be present.
```

**Pull request merged**